docs/online-schema-upgrade.md

```diff
@@ -1,13 +1,17 @@
-| RSU or Rolling Schema Upgrade| This method guarantees high availability during the schema upgrades.| The node desynchronizes with the cluster and disables flow control during the execution of the DDL statement. The rest of the cluster is not affected. After the statement execution, the node applies delayed events and synchronizes with the cluster. Although the cluster is active, during the process some nodes have the newer schema and some nodes have the older schema. The RSU method is a manual operation. For this method, the `gcache` must be large enough to store the data for the duration of the DDL change.|
-| NBO or Non-Blocking Operation| This method is used when consistency is important and uses a more efficient locking strategy.| This method is similar to `TOI`. DDL operations acquire an exclusive metadata lock on the table or schema at a late stage of the operation when updating the table or schema definition. Attempting a State Snapshot Transfer (SST) fails during the NBO operation. This mode uses a more efficient locking strategy and avoids the `TOI` issue of long-running DDL statements blocking other updates in the cluster. |
+| [TOI or Total Order Isolation]| Consistency is important. | Other transactions are blocked while the cluster processes the DDL statements. This is the default method for the [wsrep-OSU-method](https://galeracluster.com/library/documentation/mysql-wsrep-options.html#wsrep-osu-method) variable. The isolation of the DDL statement guarantees consistency. The DDL replication uses a statement format with each node processing the replicated DDL statement at same position in the replication stream. All other writes must wait until the DDL statement is executed. Long-running transactions using the same resources may receive a deadlock error at commit and be rolled back. The [pt-online-schema-change](https://www.percona.com/doc/percona-toolkit/LATEST/pt-online-schema-change.html) in the Percona Toolkit can alter the table without using locks. There are limitations: only InnoDB tables can be altered, and the `wsrep_OSU_method` must be `TOI`.|
```

Comment:
The isolation of the DDL statement guarantees consistency. The DDL replication uses a statement format with each node processing the replicated DDL statement at `the` same position in the replication stream.